Simulator/player.py

```diff
@@ -1,10 +1,14 @@
 import math
 import time
+from typing import Dict, List, Optional
+
 import numpy as np
 import random
 from Simulator import champion, origin_class
 import Simulator.utils as utils
 import Simulator.config as config
+from Simulator.augments import Augment, SelectedAugmentConfig, handle_start_round_augment, \
```

Comment:
I'm going to be getting at whoever implements items and abilities that they should probably be part of a class. At least items. I think same thing applies here.